isofit/inversion/inverse_simple.py

```diff
@@ -394,8 +409,8 @@ def invert_analytical(
     if diag_uncert:
         full_unc = np.ones(len(x))
         if fm.RT.glint_model:
-            C_rcond_idx = np.concatenate((winidx, fm.idx_surface[-2:]), axis=0)
-            full_unc[C_rcond_idx] = np.sqrt(np.diag(C_rcond))
+            # C_rcond_idx = np.concatenate((winidx, fm.idx_surface[-2:]), axis=0)
```

Comment:
Let's remove this deprecated comment.

Reply:
Removed with [b417a2b](https://github.com/isofit/isofit/pull/521/commits/b417a2bcc621588c946e9e37a3b50aeed3634dec).